conans/model/ref.py

```diff
@@ -19,10 +19,10 @@ def validate_conan_name(name):
                           " at least %s characters." % (name, ConanFileReference.min_chars)
             else:
                 message = "'%s' is an invalid name. Valid names MUST begin with a "\
-                            "letter or number, have between %s-%s chars, including "\
```

Comment:
Nop, it was just a thought, it is fine for these PRs sizes. Just remember to avoid such cleanings and refactors if the PR is big, in that case it is better to chain a couple of PRs. Thanks!